spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSettingsConfigurationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerConfig;
+import com.linecorp.armeria.server.VirtualHost;
+import com.linecorp.armeria.spring.ArmeriaSettingsConfigurationTest.TestConfiguration;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = TestConfiguration.class)
+@ActiveProfiles({ "local", "settings" })
+@DirtiesContext
+class ArmeriaSettingsConfigurationTest {
```

Comment:
Can we also have this test in the webflux module?

Reply:
Hmm, how can I test `ServerConfig`? I can't find a way to access `Server`.

Reply:
Oops, I missed this comment. 😅 
Perhaps, we can do this by:
- specifying management port `@SpringBootTest(properties = "management.server.port=0")`
- and autowiring the `ArmeriaWebServer`? `@Autowired ArmeriaWebServer webServer;`

Reply:
Thanks! I injected `Server` since `ArmeriaWebServer` registers `Server` as bean.
https://github.com/line/armeria/blob/cb23adef71f4a819a52a1017984baa79db930b8b/spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaWebServer.java#L93

Reply:
Oops, I didn't know about it. Thanks! 😄